public/locales/pt/translation.json

```
@@ -52,7 +53,7 @@
     "importBackendFailed": "O carregamento no servidor falhou.",
     "cannotExportEmptyCanvas": "Não é possível exportar um canvas vazío.",
     "couldNotCopyToClipboard": "Não foi possível copiar no clipboard. Experimente no navegador Chrome.",
-    "copiedToClipboard": "Copiado no clipboard: {{url}}"
+    "urlCopiedClipboardSuccess": "Link foi copiado para área de transferência"
```

```suggestion
    "urlCopiedClipboardSuccess": "O link foi copiado no clipboard"
```

I prefer using clipboard since "área de transferência" might mean "download section"